src/components/view.js

```diff
@@ -1,35 +1,98 @@
+
     li.classList.add('search__suggestion');
-    li.dataset.key = `${suggestion._id}`;
-    li.innerHTML = `${suggestion.area_type}, ${suggestion.city}, ${suggestion.state_code}`;
-    SUGGESTION_LIST.appendChild(li);
+    li.innerHTML = `${area_type}, ${city}, ${state_code}`;
+    suggestionList.appendChild(li);
   }
-  if (state.showSuggestions) {
-    SUGGESTION_LIST.style.height =
-      (state.suggestions.length > 4 ? 4 * 56 : state.suggestions.length * 56) +
-      'px';
+
+  if (showSuggestions) {
+    suggestionList.style.height =
+      suggestions.slice(0, suggestionToShow).length * suggestionHeight + 'px';
   }
 }
 
 export function renderSearchList() {
-  while (RECENT_SEARCHES_LIST.firstChild) {
-    RECENT_SEARCHES_LIST.removeChild(RECENT_SEARCHES_LIST.firstChild);
+  while (recentSearchList.firstChild) {
+    recentSearchList.removeChild(recentSearchList.firstChild);
   }
-  for (let search of state.recentSearches) {
+
+  const { recentSearches } = state;
+
+  for (const search of recentSearches) {
+    const { city, area_type } = search;
     const li = document.createElement('li');
     li.classList.add('recent-searches__item');
     const span = document.createElement('span');
     li.appendChild(span);
-    span.innerHTML = `Search #${state.recentSearches.indexOf(search) + 1} ${
-      search.area_type
-    }, ${search.city}`;
-    RECENT_SEARCHES_LIST.appendChild(li);
+    span.innerHTML = `Search #${recentSearches.indexOf(search) + 1} ${area_type}, ${city}`;
+    recentSearchList.appendChild(li);
+  }
+}
+
+export function renderProperties() {
+  if (state.properties.length - state.propertyOffset > 20) loadMoreButton.style.display = 'block';
+  else loadMoreButton.style.display = 'none';
+
+  const toRender = 20;
+  let properties = state.properties.slice(state.propertyOffset, state.propertyOffset + toRender);
+  matchesCount.innerHTML = `${state.propertyOffset + properties.length} of ${
+    state.properties.length
+  } matches`;
+
+  for (let property of properties) {
+    const {
+      address: { city, line },
+      price,
+      thumbnail,
+    } = property;
+
+    if (!thumbnail) continue;
+
+    const li = document.createElement('li');
+    li.classList.add('property-item');
+
+    const img = document.createElement('img');
+    img.src = thumbnail;
+    img.alt = 'thumbnail';
+    img.classList.add('property-item__thumbnail');
+
+    const div = document.createElement('div');
+    div.classList.add('property-item__info');
+
+    const priceElement = document.createElement('p');
+    priceElement.classList.add('property-item__price');
+    priceElement.innerHTML = `$ ${price}`;
+
+    const locationElement = document.createElement('p');
+    locationElement.classList.add('property-item__location');
+    locationElement.innerHTML = `${city}, ${line}`;
+
+    li.appendChild(img);
+    li.appendChild(div);
+    div.appendChild(priceElement);
+    div.appendChild(locationElement);
+
+    propertyList.appendChild(li);
```

Comment:
If function more than 10-15 lines it is sign that you did something wrong, please think about refactoring such functions